tests/transforms/test_qcut.py

```diff
@@ -729,3 +747,46 @@ def test_communication_graph_persistence(self):
 
         assert communication_graph_0.nodes == communication_graph_1.nodes
         assert communication_graph_0.edges == communication_graph_1.edges
+
+
+class TestGraphToTape:
+    def test_graph_to_tape(self):
+        """
+        Tests that a directed multigraph is correctly converted to a tape
+        """
+
+        g = qcut.tape_to_graph(multi_cut_tape)
```

Comment:
Perhaps we could use a slightly tweaked version of `multi_cut_tape`. It'd be nice to have a fragment graph which contains operation nodes whose `order` attribute is non-unique. I had troubles with this before, I believe it is fixed using `key=lambda x: x[0]` in `sort()`, but good to check.

This could be achieved, for example, by introducing a multi-wire `WireCut`, e.g., by changing the first wire cut in `multi_cut_tape` to be `qml.WireCut(wires=[0, "a"])`.